test/test_fx.py

```diff
@@ -55,7 +70,16 @@ def forward(self, A, b=4, *args, c=5, **kwargs):
                 return x
 
         t = T()
-        symbolic_trace(t)
+        s = symbolic_trace(t)
```

Comment:
Is this value `s` supposed to be used somewhere?

Reply:
oops, left this in